src/test/resources/org/broadinstitute/hellbender/tools/haplotypecaller/expected.testGVCFMode.gatk4.alleleSpecific.g.vcf

```diff
@@ -1316,73 +1316,73 @@
 20	10099191	.	C	<NON_REF>	.	.	END=10099219	GT:DP:GQ:MIN_DP:PL	0/0:58:99:46:0,111,1665
-20	10099220	.	A	G,<NON_REF>	598.60	.	AS_RAW_BaseQRankSum=|-1.7,1|NaN;AS_RAW_MQ=90000.00|59477.00|0.00;AS_RAW_MQRankSum=|-2.6,1|NaN;AS_RAW_ReadPosRankSum=|0.6,1|NaN;AS_SB_TABLE=15,10|11,9|0,0;BaseQRankSum=-1.664;DP=47;ExcessHet=3.0103;MLEAC=1,0;MLEAF=0.500,0.00;MQRankSum=-2.596;RAW_MQandDP=156677,47;ReadPosRankSum=0.629	GT:AD:DP:GQ:PL:SB	0/1:25,20,0:45:99:606,0,873,681,933,1614:15,10,11,9
+20	10099220	.	A	G,<NON_REF>	598.64	.	AS_RAW_BaseQRankSum=|-1.7,1|NaN;AS_RAW_MQ=90000.00|59477.00|0.00;AS_RAW_MQRankSum=|-2.6,1|NaN;AS_RAW_ReadPosRankSum=|0.6,1|NaN;AS_SB_TABLE=15,10|11,9|0,0;BaseQRankSum=-1.664;DP=47;ExcessHet=3.0103;MLEAC=1,0;MLEAF=0.500,0.00;MQRankSum=-2.596;RAW_MQandDP=156677,47;ReadPosRankSum=0.629	GT:AD:DP:GQ:PL:SB	0/1:25,20,0:45:99:606,0,873,681,933,1614:15,10,11,9
 20	10099221	.	T	<NON_REF>	.	.	END=10099229	GT:DP:GQ:MIN_DP:PL	0/0:44:99:42:0,102,1530
 20	10099230	.	T	<NON_REF>	.	.	END=10099233	GT:DP:GQ:MIN_DP:PL	0/0:41:90:41:0,90,1350
 20	10099234	.	T	<NON_REF>	.	.	END=10099235	GT:DP:GQ:MIN_DP:PL	0/0:44:87:44:0,87,1305
 20	10099236	.	T	<NON_REF>	.	.	END=10099238	GT:DP:GQ:MIN_DP:PL	0/0:41:90:39:0,90,1350
 20	10099239	.	C	<NON_REF>	.	.	END=10099246	GT:DP:GQ:MIN_DP:PL	0/0:36:84:35:0,84,1260
 20	10099247	.	C	<NON_REF>	.	.	END=10099248	GT:DP:GQ:MIN_DP:PL	0/0:37:93:36:0,93,1395
 20	10099249	.	C	<NON_REF>	.	.	END=10099249	GT:DP:GQ:MIN_DP:PL	0/0:36:87:36:0,87,1305
-20	10099250	.	G	A,<NON_REF>	720.60	.	AS_RAW_BaseQRankSum=|3.1,1|NaN;AS_RAW_MQ=57600.00|61406.00|0.00;AS_RAW_MQRankSum=|-2.3,1|NaN;AS_RAW_ReadPosRankSum=|0.5,1|NaN;AS_SB_TABLE=12,4|11,10|0,0;BaseQRankSum=3.170;DP=39;ExcessHet=3.0103;MLEAC=1,0;MLEAF=0.500,0.00;MQRankSum=-2.272;RAW_MQandDP=126206,39;ReadPosRankSum=0.583	GT:AD:DP:GQ:PL:SB	0/1:16,21,0:37:99:728,0,424,776,487,1263:12,4,11,10
+20	10099250	.	G	A,<NON_REF>	720.64	.	AS_RAW_BaseQRankSum=|3.1,1|NaN;AS_RAW_MQ=57600.00|61406.00|0.00;AS_RAW_MQRankSum=|-2.3,1|NaN;AS_RAW_ReadPosRankSum=|0.5,1|NaN;AS_SB_TABLE=12,4|11,10|0,0;BaseQRankSum=3.170;DP=39;ExcessHet=3.0103;MLEAC=1,0;MLEAF=0.500,0.00;MQRankSum=-2.272;RAW_MQandDP=126206,39;ReadPosRankSum=0.583	GT:AD:DP:GQ:PL:SB	0/1:16,21,0:37:99:728,0,424,776,487,1263:12,4,11,10
 20	10099251	.	A	<NON_REF>	.	.	END=10099269	GT:DP:GQ:MIN_DP:PL	0/0:30:81:28:0,81,1132
-20	10099270	.	A	<NON_REF>	.	.	END=10099270	GT:DP:GQ:MIN_DP:PL	0/0:30:37:30:0,37,1019
+20	10099270	.	A	C,<NON_REF>	0	.	AS_RAW_BaseQRankSum=|-2.9,1|NaN;AS_RAW_MQ=92210.00|6058.00|0.00;AS_RAW_MQRankSum=|-2.7,1|NaN;AS_RAW_ReadPosRankSum=|-0.2,1|NaN;AS_SB_TABLE=19,8|0,3|0,0;BaseQRankSum=-2.866;DP=36;ExcessHet=3.0103;MLEAC=0,0;MLEAF=0.00,0.00;MQRankSum=-2.691;RAW_MQandDP=114878,36;ReadPosRankSum=-0.138	GT:AD:DP:GQ:PL:SB	0/0:27,3,0:30:60:0,60,1084,81,1092,1113:19,8,0,3
```

Comment:
The root change is here.  I think it's worth running through the debugger real quick.

Reply:
Thanks for pointing me there.  I will look into it.

Reply:
Okay, it's nothing special, just a value that eked over a threshold with the new prior.
Our `configuration.genotypeArgs.STANDARD_CONFIDENCE_FOR_CALLING` is `-0.0`, but we use a miniscule positive value in practice:
```
public boolean passesThreshold(final Allele allele, final double phredScaleQualThreshold) {
        Utils.nonNull(allele);
        return getLog10PosteriorOfAlleleAbsent(allele) + EPSILON < QualityUtils.qualToErrorProbLog10(phredScaleQualThreshold);
    }
```
where `EPSILON` is 10^-10.  The C alt allele at this site has, with the correct SNP prior, a probability of 7*10^-10, so it just barely makes it.

So, there's nothing amiss here.  A case could be made for increasing our `EPSILON`, but that's independent of this change.

Reply:
So many epsilons, so little time...  I'm satisfied with this explanation.  :shipit: